feature-libs/product-configurator/common/components/configure-cart-entry/configure-cart-entry.component.spec.ts

```diff
@@ -29,6 +35,20 @@ class MockUrlPipe implements PipeTransform {
   transform(): any {}
 }
 
+const mockRouterState: any = {
+  state: {
+    semanticRoute: 'home',
+  },
+};
+
+let routerStateObservable: any = null;
```

Comment:
(1) That's no observable (naming) 
(2) Please add typing where possible
```suggestion
let routerState: RouterState
```